src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,42 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: block;
+  position: relative;
+  padding-top: 23px;
+  padding-bottom: 23px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: $primary-text-color;
+}
```

Comment:
use nesting `.nav { &__ }`, also fix in other palces

Reply:
not fixed